tests/scripts/run-xunit-perf.cmd

```diff
@@ -273,15 +288,19 @@ rem ****************************************************************************
 
   set LV_MEASUREMENT_ARGS=
   set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% %BENCHVIEW_MEASUREMENT_PARSER%
-  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% "Perf-%BENCHNAME%.xml"
-  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% --better desc
-  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% --drop-first-value
+  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% --better %BETTER%
+  set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% %HAS_WARMUP_RUN%
   set LV_MEASUREMENT_ARGS=%LV_MEASUREMENT_ARGS% --append
-  call :run_cmd py.exe "%BENCHVIEW_PATH%\measurement.py" %LV_MEASUREMENT_ARGS%
-  IF %ERRORLEVEL% NEQ 0 (
-    call :print_error Failed to generate BenchView measurement data.
-    exit /b 1
+
+  for /f "delims=|" %%f in ('dir /b Perf-*.xml 2^>nul') do (
+    call :run_cmd py.exe "%BENCHVIEW_PATH%\measurement.py" %LV_MEASUREMENT_ARGS% %%f %%~nf.json
+
+    IF %ERRORLEVEL% NEQ 0 (
```

Comment:
Because of delayed expansion, this will always pass. You could change it to:
```cmd
IF !ERRORLEVEL! NEQ 0 (
```
optionally, end the previous statement with a `|| (` and remove the `IF` line.